Watching for file changes with StatReloader
Главная страница была посещена.
"GET / HTTP/1.1" 200 951
"GET /static/my_dz_1/css/style.css HTTP/1.1" 200 609
"GET /static/my_dz_1/image/583.jpg HTTP/1.1" 200 2057688
"О себе" страница была посещена.
"GET /about/ HTTP/1.1" 200 1289
Главная страница была посещена.
"GET / HTTP/1.1" 200 951
"О себе" страница была посещена.
"GET /about/ HTTP/1.1" 200 1289
Главная страница была посещена.
"GET / HTTP/1.1" 200 951
Watching for file changes with StatReloader
Главная страница была посещена.
"GET / HTTP/1.1" 200 951
"GET /static/my_dz_1/css/style.css HTTP/1.1" 304 0
Главная страница была посещена.
"GET / HTTP/1.1" 200 951
"О себе" страница была посещена.
"GET /about/ HTTP/1.1" 200 1289
Главная страница была посещена.
"GET / HTTP/1.1" 200 951
Watching for file changes with StatReloader
Главная страница была посещена.
"GET / HTTP/1.1" 200 903
"О себе" страница была посещена.
"GET /about/ HTTP/1.1" 200 1211
Главная страница была посещена.
"GET / HTTP/1.1" 200 903
"О себе" страница была посещена.
"GET /about/ HTTP/1.1" 200 1211
Главная страница была посещена.
"GET / HTTP/1.1" 200 903
"О себе" страница была посещена.
"GET /about/ HTTP/1.1" 200 1211
"О себе" страница была посещена.
"GET /about/ HTTP/1.1" 200 1211
C:\Users\it\Desktop\seminar_dz\seminar_dz\my_dz_1\views.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Главная страница была посещена.
"GET / HTTP/1.1" 200 903
Главная страница была посещена.
"GET / HTTP/1.1" 200 903
"О себе" страница была посещена.
"GET /about/ HTTP/1.1" 200 903
Главная страница была посещена.
"GET / HTTP/1.1" 200 903
"О себе" страница была посещена.
"GET /about/ HTTP/1.1" 200 903
"О себе" страница была посещена.
"GET /about/ HTTP/1.1" 200 903
Главная страница была посещена.
"GET / HTTP/1.1" 200 903
"О себе" страница была посещена.
"GET /about/ HTTP/1.1" 200 903
C:\Users\it\Desktop\seminar_dz\seminar_dz\my_dz_1\views.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Главная страница была посещена.
"GET / HTTP/1.1" 200 903
"GET /static/my_dz_1/css/style.css HTTP/1.1" 304 0
"О себе" страница была посещена.
"GET /about/ HTTP/1.1" 200 1213
Главная страница была посещена.
"GET / HTTP/1.1" 200 903
"О себе" страница была посещена.
"GET /about/ HTTP/1.1" 200 1213
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Главная страница была посещена.
Internal Server Error: /
Traceback (most recent call last):
  File "C:\Users\it\Desktop\seminar_dz\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\it\Desktop\seminar_dz\.venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\it\Desktop\seminar_dz\seminar_dz\my_dz_1\views.py", line 13, in index
    return render(request, 'my_dz_1/index.html', {'title': 'Главная страница'})
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\it\Desktop\seminar_dz\.venv\Lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\it\Desktop\seminar_dz\.venv\Lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\it\Desktop\seminar_dz\.venv\Lib\site-packages\django\template\loader.py", line 19, in get_template
    raise TemplateDoesNotExist(template_name, chain=chain)
django.template.exceptions.TemplateDoesNotExist: my_dz_1/index.html
"GET / HTTP/1.1" 500 78889
Not Found: /favicon.ico
"GET /favicon.ico HTTP/1.1" 404 2442
Watching for file changes with StatReloader
"GET / HTTP/1.1" 200 10629
Watching for file changes with StatReloader
Not Found: /
"GET / HTTP/1.1" 404 2462
Not Found: /favicon.ico
"GET /favicon.ico HTTP/1.1" 404 2513
C:\Users\it\Desktop\seminar_dz\seminar_dz\my_dz_2\urls.py changed, reloading.
Watching for file changes with StatReloader
Not Found: /
"GET / HTTP/1.1" 404 2261
"GET /client/2 HTTP/1.1" 301 0
Internal Server Error: /client/2/
Traceback (most recent call last):
  File "C:\Users\it\Desktop\seminar_dz\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\it\Desktop\seminar_dz\.venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\it\Desktop\seminar_dz\seminar_dz\my_dz_2\views.py", line 7, in client_detail
    client = Client.objects.get(pk=client_id)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\it\Desktop\seminar_dz\.venv\Lib\site-packages\django\db\models\manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\it\Desktop\seminar_dz\.venv\Lib\site-packages\django\db\models\query.py", line 649, in get
    raise self.model.DoesNotExist(
my_dz_2.models.Client.DoesNotExist: Client matching query does not exist.
"GET /client/2/ HTTP/1.1" 500 71682
Not Found: /client/<int:client_id 2
"GET /client/%3Cint:client_id%202 HTTP/1.1" 404 2358
Not Found: /client/client_id 2
"GET /client/client_id%202 HTTP/1.1" 404 2335
Not Found: /client/client_id
"GET /client/client_id HTTP/1.1" 404 2327
Not Found: /client/client_id
"GET /client/client_id?2 HTTP/1.1" 404 2329
Not Found: /client/
"GET /client/ HTTP/1.1" 404 2300
"GET /client/1 HTTP/1.1" 301 0
Internal Server Error: /client/1/
Traceback (most recent call last):
  File "C:\Users\it\Desktop\seminar_dz\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\it\Desktop\seminar_dz\.venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\it\Desktop\seminar_dz\seminar_dz\my_dz_2\views.py", line 9, in client_detail
    return render(request, 'client_detail.html', {'client': client, 'orders': orders})
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\it\Desktop\seminar_dz\.venv\Lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\it\Desktop\seminar_dz\.venv\Lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\it\Desktop\seminar_dz\.venv\Lib\site-packages\django\template\loader.py", line 19, in get_template
    raise TemplateDoesNotExist(template_name, chain=chain)
django.template.exceptions.TemplateDoesNotExist: client_detail.html
"GET /client/1/ HTTP/1.1" 500 78733
Internal Server Error: /client/1/
Traceback (most recent call last):
  File "C:\Users\it\Desktop\seminar_dz\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\it\Desktop\seminar_dz\.venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\it\Desktop\seminar_dz\seminar_dz\my_dz_2\views.py", line 9, in client_detail
    return render(request, 'client_detail.html', {'client': client, 'orders': orders})
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\it\Desktop\seminar_dz\.venv\Lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\it\Desktop\seminar_dz\.venv\Lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\it\Desktop\seminar_dz\.venv\Lib\site-packages\django\template\loader.py", line 19, in get_template
    raise TemplateDoesNotExist(template_name, chain=chain)
django.template.exceptions.TemplateDoesNotExist: client_detail.html
"GET /client/1/ HTTP/1.1" 500 78733
Internal Server Error: /client/1/
Traceback (most recent call last):
  File "C:\Users\it\Desktop\seminar_dz\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\it\Desktop\seminar_dz\.venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\it\Desktop\seminar_dz\seminar_dz\my_dz_2\views.py", line 9, in client_detail
    return render(request, 'client_detail.html', {'client': client, 'orders': orders})
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\it\Desktop\seminar_dz\.venv\Lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\it\Desktop\seminar_dz\.venv\Lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\it\Desktop\seminar_dz\.venv\Lib\site-packages\django\template\loader.py", line 19, in get_template
    raise TemplateDoesNotExist(template_name, chain=chain)
django.template.exceptions.TemplateDoesNotExist: client_detail.html
"GET /client/1/ HTTP/1.1" 500 78870
Internal Server Error: /client/1/
Traceback (most recent call last):
  File "C:\Users\it\Desktop\seminar_dz\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\it\Desktop\seminar_dz\.venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\it\Desktop\seminar_dz\seminar_dz\my_dz_2\views.py", line 9, in client_detail
    return render(request, 'client_detail.html', {'client': client, 'orders': orders})
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\it\Desktop\seminar_dz\.venv\Lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\it\Desktop\seminar_dz\.venv\Lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\it\Desktop\seminar_dz\.venv\Lib\site-packages\django\template\loader.py", line 19, in get_template
    raise TemplateDoesNotExist(template_name, chain=chain)
django.template.exceptions.TemplateDoesNotExist: client_detail.html
"GET /client/1/ HTTP/1.1" 500 78870
Internal Server Error: /client/1/
Traceback (most recent call last):
  File "C:\Users\it\Desktop\seminar_dz\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\it\Desktop\seminar_dz\.venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\it\Desktop\seminar_dz\seminar_dz\my_dz_2\views.py", line 9, in client_detail
    return render(request, 'client_detail.html', {'client': client, 'orders': orders})
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\it\Desktop\seminar_dz\.venv\Lib\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\it\Desktop\seminar_dz\.venv\Lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\it\Desktop\seminar_dz\.venv\Lib\site-packages\django\template\loader.py", line 19, in get_template
    raise TemplateDoesNotExist(template_name, chain=chain)
django.template.exceptions.TemplateDoesNotExist: client_detail.html
"GET /client/1/ HTTP/1.1" 500 78870
C:\Users\it\Desktop\seminar_dz\seminar_dz\my_dz_2\views.py changed, reloading.
Watching for file changes with StatReloader
"GET /client/1/ HTTP/1.1" 200 540
Internal Server Error: /client/2/
Traceback (most recent call last):
  File "C:\Users\it\Desktop\seminar_dz\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\it\Desktop\seminar_dz\.venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\it\Desktop\seminar_dz\seminar_dz\my_dz_2\views.py", line 7, in client_detail
    client = Client.objects.get(pk=client_id)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\it\Desktop\seminar_dz\.venv\Lib\site-packages\django\db\models\manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\it\Desktop\seminar_dz\.venv\Lib\site-packages\django\db\models\query.py", line 649, in get
    raise self.model.DoesNotExist(
my_dz_2.models.Client.DoesNotExist: Client matching query does not exist.
"GET /client/2/ HTTP/1.1" 500 71690
"GET /client/3/ HTTP/1.1" 200 533
"GET /client/3/ HTTP/1.1" 200 615
"GET /static/my_dz_1/css/style.css HTTP/1.1" 404 1822
"GET /client/3/ HTTP/1.1" 200 615
"GET /static/my_dz_1/css/style.css HTTP/1.1" 404 1822
"GET /client/3/ HTTP/1.1" 200 615
"GET /static/my_dz_1/css/style.css HTTP/1.1" 404 1822
"GET /client/3/ HTTP/1.1" 200 615
"GET /static/my_dz_1/css/style.css HTTP/1.1" 404 1822
"GET /client/3/ HTTP/1.1" 200 615
"GET /static/my_dz_2/css/style.css HTTP/1.1" 200 609
"GET /client/3/ HTTP/1.1" 200 615
"GET /client/3/ HTTP/1.1" 200 664
"GET /client/3/ HTTP/1.1" 200 1610
Watching for file changes with StatReloader
Watching for file changes with StatReloader
Not Found: /
"GET / HTTP/1.1" 404 2261
Internal Server Error: /client/3/
Traceback (most recent call last):
  File "C:\Users\it\Desktop\seminar_dz\.venv\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\it\Desktop\seminar_dz\.venv\Lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\it\Desktop\seminar_dz\seminar_dz\my_dz_2\views.py", line 7, in client_detail
    client = Client.objects.get(pk=client_id)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\it\Desktop\seminar_dz\.venv\Lib\site-packages\django\db\models\manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\it\Desktop\seminar_dz\.venv\Lib\site-packages\django\db\models\query.py", line 649, in get
    raise self.model.DoesNotExist(
my_dz_2.models.Client.DoesNotExist: Client matching query does not exist.
"GET /client/3/ HTTP/1.1" 500 71708
Watching for file changes with StatReloader
"GET /client/1/ HTTP/1.1" 200 1213
Watching for file changes with StatReloader
"GET /client/1/ HTTP/1.1" 200 1616
"GET /client/2/ HTTP/1.1" 200 762
"GET /client/3/ HTTP/1.1" 200 761
Watching for file changes with StatReloader
Not Found: /
"GET / HTTP/1.1" 404 2261
Not Found: /favicon.ico
"GET /favicon.ico HTTP/1.1" 404 2312
"GET /client/3/ HTTP/1.1" 200 761
"GET /static/my_dz_2/css/style.css HTTP/1.1" 304 0
"GET /client/1/ HTTP/1.1" 200 1616
Watching for file changes with StatReloader
Not Found: /
"GET / HTTP/1.1" 404 2261
"GET /client/3/ HTTP/1.1" 200 761
"GET /static/my_dz_2/css/style.css HTTP/1.1" 304 0
"GET /client/1/ HTTP/1.1" 200 1616
C:\Users\it\Desktop\seminar_dz\seminar_dz\my_dz_2\models.py changed, reloading.
Watching for file changes with StatReloader
Watching for file changes with StatReloader
"GET /client/4/ HTTP/1.1" 200 760
C:\Users\it\Desktop\seminar_dz\seminar_dz\my_dz_2\views.py changed, reloading.
Watching for file changes with StatReloader
"GET /clients HTTP/1.1" 301 0
"GET /clients/ HTTP/1.1" 200 737
Watching for file changes with StatReloader
"GET /clients/ HTTP/1.1" 200 737
"GET /clients/ HTTP/1.1" 200 737
C:\Users\it\Desktop\seminar_dz\seminar_dz\my_dz_2\views.py changed, reloading.
Watching for file changes with StatReloader
"GET /clients/ HTTP/1.1" 200 690
"GET /clients/ HTTP/1.1" 200 690
"GET /client/4/ HTTP/1.1" 200 760
"GET /clients/ HTTP/1.1" 200 690
C:\Users\it\Desktop\seminar_dz\seminar_dz\my_dz_2\views.py changed, reloading.
Watching for file changes with StatReloader
"GET /clients/ HTTP/1.1" 200 737
"GET /clients/ HTTP/1.1" 200 737
"GET /clients/ HTTP/1.1" 200 796
"GET /client/2/ HTTP/1.1" 200 762
"GET /client/4/ HTTP/1.1" 200 760
"GET /client/4/ HTTP/1.1" 200 760
"GET /client/3/ HTTP/1.1" 200 761
"GET /client/4/ HTTP/1.1" 200 760
"GET /client/3/ HTTP/1.1" 200 761
"GET /client/2/ HTTP/1.1" 200 762
"GET /client/1/ HTTP/1.1" 200 1616
"GET /client/2/ HTTP/1.1" 200 762
Watching for file changes with StatReloader
Not Found: /
"GET / HTTP/1.1" 404 2436
"GET /client/3/ HTTP/1.1" 200 761
"GET /clients/ HTTP/1.1" 200 796
"GET /client/2/ HTTP/1.1" 200 762
"GET /client/1/ HTTP/1.1" 200 1616
"GET /client/4/ HTTP/1.1" 200 760
Not Found: /clients/3
"GET /clients/3 HTTP/1.1" 404 2481
"GET /client/3 HTTP/1.1" 301 0
"GET /client/3/ HTTP/1.1" 200 761
"GET /client/1/ HTTP/1.1" 200 1616
Not Found: /
"GET / HTTP/1.1" 404 2436
"GET /client/3/ HTTP/1.1" 200 761
